### PR TITLE
#5806 Begin styled article page

### PR DESCRIPTION
**Pull request by Flame787898**
https://redmine.devbranch.work/issues/5806

### CODE REVIEW

File: web/themes/custom/green_theme/webpack.config.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ module.exports = {
     new BrowserSyncPlugin({
       host: "localhost",
       port: 3000,
-      proxy: "http://drupal.docker.localhost:8000"
+      proxy: "http://greendddd.docksal/"
```
Comment (Y0r) [inline web/themes/custom/green_theme/webpack.config.js:33]: not sure

File: web/themes/custom/green_theme/templates/node--posts--full.html.twig [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    </div>
+    <div class="node__tags">
+      {{ content.field_tag}}
+    </div>
+    <div class="wrap-social">
+      <div class="icon"><a href="https://www.facebook.com/" target="_blank">1<i class="fa-brands fa-facebook-f"></i></a></div>
+      <div class="icon"><a href="https://www.instagram.com/" target="_blank">2<i class="fa-brands fa-instagram"></i></a></div>
+      <div class="icon"><a href="https://twitter.com/?lang=en" target="_blank">3<i class="fa-brands fa-twitter"></i></a></div>
+      <div class="icon"><a href="https://www.pinterest.com/" target="_blank">4<i class="fa-brands fa-pinterest-p"></i></a></div>
+    </div>
```
Comment (Y0r) [inline web/themes/custom/green_theme/templates/node--posts--full.html.twig:110]: Hardcoded content

**Comment by Flame787898** [comment]
https://redmine.devbranch.work/issues/5806

**Review by Y0r — APPROVED** [review]

File: web/sites/default/settings.php [hunk 3]
```diff
@@ -795,11 +795,11 @@
...
-  'password' => 'drupal',
+  'database' => 'default',
+  'username' => 'user',
+  'password' => 'user',
   'prefix' => '',
-  'host' => 'mariadb',
+  'host' => 'db',
   'port' => '3306',
   'namespace' => 'Drupal\\mysql\\Driver\\Database\\mysql',
   'driver' => 'mysql',
```
Comment (Y0r) [inline web/sites/default/settings.php:805]: settings.php should be under commit

**Review by Y0r — APPROVED** [review]

**Pull request merged**